cli/config/config.go

```diff
@@ -84,8 +84,14 @@ func LoadFromReader(configData io.Reader) (*configfile.ConfigFile, error) {
 	return &configFile, err
 }
 
-// Load reads the configuration files in the given directory, and sets up
-// the auth config information and returns values.
+// Load reads the configuration file ([ConfigFileName]) from the given directory.
+// If no directory is given, it uses the default [Dir]. A [configfile.ConfigFile]
```

Comment:
nit: `[*configfile.ConfigFile]` here since it's a pointer?

Reply:
Oh! Looks like that works as well; I thought GoDoc links wouldn't accept that (so requiring to manually construct a link), but it looks like it does;

<img width="998" alt="Screenshot 2024-05-22 at 10 20 03" src="https://github.com/docker/cli/assets/1804568/09182415-15d1-45a5-9748-089622eef339">